bin/filter_sce_rds.R

```diff
@@ -20,6 +20,13 @@ option_list <- list(
     type = "character",
     help = "path to output filtered rds file. Must end in .rds"
   ),
+  make_option(
+    opt_str = c("--prob_compromised_cutoff"),
+    type = "double",
+    default = 0.75,
+    help = "probability compromised cutoff used for filtering cells with miQC",
+    metavar = "double"
```

Comment:
Shouldn't need the metavar, but if we do want it, it should probably be something like `PROB`. (This is the text that is displayed with help in place of the value that would be set, like `--prob_compromized_cutoff PROB`, but we can leave it off.
```suggestion
    help = "probability compromised cutoff used for filtering cells with miQC"
```